contrib/babelfishpg_tds/src/backend/tds/tdsbulkload.c

```diff
@@ -233,10 +233,12 @@ GetBulkLoadRequest(StringInfo message)
 	{
 		CheckMessageHasEnoughBytesToRead(&message, COLUMNMETADATA_HEADER_LEN);
 		/* UserType */
+		Assert(sizeof(uint32_t)<=sizeof(colmetadata[currentColumn].userType));
```

Comment:
This seems pointless too. Can you explain why we need an assert after CheckMessageHasEnoughBytesToRead?
CheckMessageHasEnoughBytesToRead will raise error if we do not have enough bytes to read.

Reply:
I added this assert for this memcpy(&colmetadata[currentColumn].userType, &message->data[offset], sizeof(uint32_t)); which is on the line 237.

Reply:
I added the comment for the same line

Reply:
Okay.